backend/src/api/item-create-or-update.ts

```diff
@@ -46,64 +54,105 @@ export const itemCreateOrUpdate = route
         checkBoardAPIAccess(request, async (board) => {
             const { itemId } = request.routeParams
             let {
+                x,
+                y,
                 type,
                 text,
                 color,
                 container,
+                width,
+                height,
+                replaceXIfExists,
+                replaceYIfExists,
                 replaceTextIfExists,
                 replaceColorIfExists,
                 replaceContainerIfExists = true,
+                replaceWidthIfExists,
+                replaceHeightIfExists,
             } = request.body
             console.log(`PUT item for board ${board.board.id} item ${itemId}: ${JSON.stringify(request.req.body)}`)
             const existingItem = board.board.items[itemId]
             if (existingItem) {
                 updateItem(
                     board,
+                    x !== undefined ? x : existingItem.x,
```

Comment:
These expressions could be simplified to `x ?? existingItem.x` for brevity.

Reply:
thought about it later, will modify here and elsewhere